apps/react-app/src/common-app/helpers/download-md-section.tsx

```diff
@@ -0,0 +1,11 @@
+export const download = (blob: Blob, filename: string) => {
```

Comment:
This file should move to common (can be reused by other application), and must be renamed to download-file-helper

we will apply this on develop